tool/site/server/src/routes/dispense/claim.ts

```diff
@@ -31,5 +36,18 @@ export default [
         response_xml: claimResponse.spineResponse
       }).code(200)
     }
+  },
+  {
+    method: "GET",
+    path: "/claim/{prescriptionId}",
```

Comment:
should this match /dispense/claim for consistency, the difference being GET vs. POST?

Reply:
POST `/dispense/claim` is the route that sends a claim to EPS API, GET `/claim/<id>` is the route to get the claims in session. this matches the dispense routes